tests/testStreamOpen.cpp

```diff
@@ -198,3 +198,14 @@ TEST_F(StreamOpen, PlaybackFormatI16ReturnsI16){
     ASSERT_EQ(mStream->getFormat(), AudioFormat::I16);
     closeStream();
 }
+
+TEST_F(StreamOpen, LowLatencyStreamHasBufferSizeOfTwoBursts){
+
+    if (mBuilder.isAAudioRecommended()){
+        mBuilder.setDirection(Direction::Output);
+        mBuilder.setPerformanceMode(PerformanceMode::LowLatency);
+        openStream();
+        ASSERT_EQ(mStream->getBufferSizeInFrames(), mStream->getFramesPerBurst() * 2);
```

Comment:
setBufferSizeInFrames() does not work for OpenSL ES callbacks. But we set the buffer size to two bursts by default. So this may happen to work.

Reply:
The reason I haven't merged this is because I couldn't get the test to pass for OpenSL ES. Need to spend more time investigating.